CS02-React&Redux.md

```diff
@@ -26,9 +26,13 @@
 - Facebook
 - Community
     - React: 1240 Contributors
+    - Around 30% of the contributors are from Facebook
```

Comment:
Interesting! I wonder what it would be by volume. For example are the 70% from outside Facebook making little small changes but the bulk of the work is from facebook. How many commits are from Facebook employees? You don't have to research this, I'm just posing the questions as food for thought.